waiter/src/waiter/handler.clj

```diff
@@ -246,14 +246,16 @@
               retrieve-instance-counts (fn retrieve-instance-counts [service-id]
                                          {:healthy-instances (count (get-in current-state [:service-id->healthy-instances service-id]))
                                           :unhealthy-instances (count (get-in current-state [:service-id->unhealthy-instances service-id]))})
+              service-id->metrics (service-id->metrics-fn)
               include-effective-parameters? (utils/request-flag request-params "effective-parameters")
               response-data (map
                              (fn service-id->service-info [service-id]
                                (let [service-description (service-id->service-description-fn service-id :effective? false)]
                                  (cond->
-                                     {:service-id service-id
+                                     {:instance-counts (retrieve-instance-counts service-id)
+                                      :last-request-time (get-in service-id->metrics [service-id "last-request-time"])
```

Comment:
Metrics are for external consumption.  We'd do well to move away using them to pass data around inside Waiter.  Would it be better to read from the agent instead?